### PR TITLE
fix(TBD-6923): Fix hive query name for tHiveRow

### PR DESCRIPTION
**Pull request by achursin-talend**
**What is the current behavior?** (You can also link to an open issue here)
https://jira.talendforge.org/browse/TBD-6923

**What is the new behavior?**

Error "Error while processing statement: hive configuration hive.query.name does not exists." does not appear

**Please check if the PR fulfills these requirements**

- [x] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.designer.components.localprovider/components/tHiveRow/tHiveRow_main.javajet [hunk 1]
```diff
@@ -187,7 +187,18 @@ if(hiveDistrib.isExecutedThroughWebHCat()) {
...
-		<%@ include file="@{org.talend.designer.components.localprovider}/components/templates/Hive/SetQueryName.javajet"%>
+		<%
+		if(hiveDistrib.doSupportTezForHive()) {
+		%>
+			<%@ include file="@{org.talend.designer.components.localprovider}/components/templates/Hive/SetQueryName.javajet"%>
+		<%
+		} else {
+		%>
+			java.text.DateFormat dateStrFormat = new java.text.SimpleDateFormat("yyyyMMddHHmmss");
+			final String queryIdentifier = projectName + "_" + jobName + "_" + jobVersion.replace(".", "_") + "_<%=cid%>_" + dateStrFormat.format(new Date(startTime));
```
Comment (rdubois) [inline main/plugins/org.talend.designer.components.localprovider/components/tHiveRow/tHiveRow_main.javajet]: The generated code wil not compile if you have 2 tHiveRow in the same subjob... Please make use of the `_<%=cid%>` to name your variables.

Comment (rdubois) [reply main/plugins/org.talend.designer.components.localprovider/components/tHiveRow/tHiveRow_main.javajet]: And BTW, where does this code come from ?

Comment (achursin-talend) [reply main/plugins/org.talend.designer.components.localprovider/components/tHiveRow/tHiveRow_main.javajet]: This code is from SetQueryName.javajet. queryIdentifier is written to console log when job is submitted to HiveServer

Comment (achursin-talend) [reply main/plugins/org.talend.designer.components.localprovider/components/tHiveRow/tHiveRow_main.javajet]: So when SetQueryName.javajet is not imported, queryIdentifier should be built manually

Comment (achursin-talend) [reply main/plugins/org.talend.designer.components.localprovider/components/tHiveRow/tHiveRow_main.javajet]: Also I've tested the subjob with 2 tHiveRow components and it compiles and runs successfully.

**Pull request closed without merging**